config/config.default.js

```diff
@@ -228,15 +229,15 @@ module.exports = appInfo => {
    * @property {Boolean} keepAlive - Enable http keepalive or not, default is true
    * @property {Number} freeSocketKeepAliveTimeout - socket keepalive max free time, default is 4000 ms.
    * @property {Number} timeout - socket max unative time, default is 30000 ms.
-   * @property {Number} maxSockets - max socket number of one host, default is Infinity.
+   * @property {Number} maxSockets - max socket number of one host, default is MAX_SAFE_INTEGER @ses https://github.com/node-modules/utility/blob/master/lib/number.js#L5.
    * @property {Number} maxFreeSockets - max free socket number of one host, default is 256.
    * @property {Boolean} enableDNSCache - Enable DNS lookup from local cache or not, default is false.
    */
   config.httpclient = {
     keepAlive: true,
     freeSocketKeepAliveTimeout: 4000,
     timeout: 30000,
-    maxSockets: Infinity,
+    maxSockets: utility.MAX_SAFE_INTEGER,
```

Comment:
I think the more proper way is to use `Number. MAX_SAFE_INTEGER` instead of `utility.MAX_SAFE_INTEGER`. We only support node.js >= 6.0, so the `Number.MAX_SAFE_INTEGER` is always available.